### PR TITLE
Issue/317 deprecate dashboard and web console section

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

Previous [PR](https://github.com/inmanta/inmanta-ui/pull/330) was based off wrong branch

- Move ui related options from inmanta-core to inmanta-ui
- Log a deprecation warning when the web-console or the dashboard sections are used 

part of #317 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Type annotations are present
- [x] Code is clear and sufficiently documented
- [x] No (preventable) type errors
- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Correct, in line with design
- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )

# Reviewer Checklist:

- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)
- [ ] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: src/inmanta_ui/config.py [hunk 1]
```diff
@@ -16,20 +16,112 @@
...
+    is_str,
+    predecessor_option=dash_auth_url,
+)
+web_ui_dashboard_client_id = Option(
+    "web-ui",
+    "dashboard_client_id",
+    None,
+    "The client id configured in keycloak for this application.",
+    is_str,
+    predecessor_option=dash_client_id,
```
Comment (arnaudsjs) [inline src/inmanta_ui/config.py:126]: I don't think this is necessary actually. The dashboard will be removed in ISO6, so it doesn't make much sense to migrate the options to a different section. Do you agree?

**Review by arnaudsjs — CHANGES_REQUESTED** [review]

File: src/inmanta_ui/config.py (hunk 1 above)
Comment (arnaudsjs) [reply src/inmanta_ui/config.py:126]: You can ignore this comment. We have to migrate the authentication-related option. So then it makes sense to migrate the other options as well.

File: src/inmanta_ui/config.py [hunk 2]
```diff
@@ -16,20 +16,116 @@
...
-web_console_enabled = Option("web-console", "enabled", True, "Should the server should host the web-console or not", is_bool)
+#######################################
+# OLD web-console configuration options
+#######################################
+
+web_console_enabled = Option(
+    "web-console",
+    "enabled",
+    True,
+    "[DEPRECATED USE :inmanta.config:option:`web-ui.console_enabled`] " "Whether the server should host the web-console or not",
```
Comment (Hugo-Inmanta) [inline src/inmanta_ui/config.py:29]: I'm not sure I have check how those are rendering.
Checking right now

Comment (Hugo-Inmanta) [reply src/inmanta_ui/config.py]: a4353c9e0745aabc23f8fc8d2ab199c7191af798 makes sure it's rendered properly

**Review by arnaudsjs — APPROVED** [review]

**Pull request closed without merging**